apps/review-host/check.sh

```
@@ -0,0 +1,130 @@
+# Procedures the mobile app calls on a host client. apiClient calls are excluded:
+# host.relayEndpoint and github.listPullRequests are cloud API, not host-service.
+NS=$(grep -oE "^[[:space:]]+[a-zA-Z][a-zA-Z0-9]*:" "$ROOT/packages/host-service/src/trpc/router/router.ts" \
+  | sed -E 's/^[[:space:]]+//; s/:$//' | sort -u | paste -sd'|' -)
+PROCS=$(grep -rhE "\b($NS)\.[a-zA-Z][a-zA-Z0-9]*\.(query|mutate)\b" "$ROOT/apps/mobile" --include="*.ts" --include="*.tsx" 2>/dev/null \
+  | grep -v apiClient \
+  | grep -oE "\b($NS)\.[a-zA-Z][a-zA-Z0-9]*\.(query|mutate)\b" | sed -E 's/\.(query|mutate)$//' | sort -u)
+COUNT=$(printf '%s\n' "$PROCS" | grep -c . || true)
+if [ -n "$RUNNING_VERSION" ] && [ "$COUNT" -gt 0 ]; then
+  # Reports per procedure so a probe that never ran is distinguishable
+  # from one that found nothing missing. -m bounds a hung procedure.
+  PROBED=$(ssh_box "sudo sh -c 'for P in $(printf '%s ' $PROCS); do R=\$(curl -s -m 15 -H \"Authorization: Bearer $SECRET\" \"http://127.0.0.1:48800/trpc/\$P\" | head -c 200); case \"\$R\" in *NOT_FOUND*) echo \"missing \$P\";; *) echo \"ok \$P\";; esac; done'")
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)